pkg/netflow/flowaggregator/aggregator_test.go

```diff
@@ -295,8 +295,10 @@ func TestAggregator_withMockPayload(t *testing.T) {
 	sender.AssertMetric(t, "Gauge", "datadog.netflow.aggregator.port_rollup.new_store_size", 4, "", nil)
 	sender.AssertMetric(t, "Gauge", "datadog.netflow.aggregator.input_buffer.capacity", 20, "", nil)
 	sender.AssertMetric(t, "Gauge", "datadog.netflow.aggregator.input_buffer.length", 0, "", nil)
+	sender.AssertMetric(t, "Count", "datadog.netflow.aggregator.sequence.delta", 6, "", []string{"exporter_ip:127.0.0.1", "device_namespace:default", "flow_type:netflow5"})
+	sender.AssertMetric(t, "Gauge", "datadog.netflow.aggregator.sequence.last", 6, "", []string{"exporter_ip:127.0.0.1", "device_namespace:default", "flow_type:netflow5"})
 	sender.AssertMetric(t, "MonotonicCount", "datadog.netflow.decoder.messages", 1, "", []string{"collector_type:netflow5", "worker:0"})
-	sender.AssertMetric(t, "MonotonicCount", "datadog.netflow.processor.flows", 1, "", []string{"exporter_ip:127.0.0.1", "version:5", "flow_protocol:netflow"})
+	sender.AssertMetric(t, "MonotonicCount", "datadog.netflow.processor.processed", 1, "", []string{"exporter_ip:127.0.0.1", "version:5", "flow_protocol:netflow"})
```

Comment:
Note: the metrics is not actually flow but UDP packets processed by goflow2 library. This new name `datadog.netflow.processor.processed` might be more appropriate.